core/src/main/java/com/graphhopper/routing/util/countryrules/europe/AlbaniaCountryRule.java

```diff
@@ -0,0 +1,36 @@
+/*
+ *  Licensed to GraphHopper GmbH under one or more contributor
+ *  license agreements. See the NOTICE file distributed with this work for
+ *  additional information regarding copyright ownership.
+ *
+ *  GraphHopper GmbH licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except in
+ *  compliance with the License. You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.graphhopper.routing.util.countryrules.europe;
+
+import com.graphhopper.reader.ReaderWay;
+import com.graphhopper.routing.ev.Toll;
+import com.graphhopper.routing.util.TransportationMode;
+import com.graphhopper.routing.util.countryrules.CountryRule;
+
+public class AlbaniaCountryRule implements CountryRule {
+    
+    @Override
+    public Toll getToll(ReaderWay readerWay, TransportationMode transportationMode, Toll currentToll) {
+        if (!transportationMode.isMotorVehicle() || currentToll != Toll.MISSING) {
+            return currentToll;
+        }
```

Comment:
Does the Toll EV depend on the TransportationMode or not? Can we not just interpret the Toll EV as representing the toll for motorized vehicles (and assume there is no such things for non-motorized vehicles)? And if yes, can we remove the transportation mode parameter from this method already? I will adjust the signatures further as we discussed, but since the `getToll` method is added here so many times can we remove the transportation mode parameter here already?

Reply:
Agreed